tests/pd/test_rpc_client.rs

```diff
@@ -11,23 +11,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::env;
+use std::thread;
+use std::sync::Arc;
+use std::time::Duration;
 
 use kvproto::metapb;
 
-use tikv::pd::{PdClient, RpcClient};
+use tikv::pd::{PdClient, RpcClient, validate_endpoints};
+
+use super::mock::case::*;
+use super::mock::Server as MockServer;
 
 #[test]
 fn test_rpc_client() {
-    // We need to set all members in PD_ENDPOINTS to pass this test.
-    let endpoints = match env::var("PD_ENDPOINTS") {
-        Ok(v) => v,
-        Err(_) => return,
-    };
+    let eps = "http://127.0.0.1:3079".to_owned();
```

Comment:
I don't think it will conflict in CI.
Besides, "listen then get address" is too complicated. We have to pass an impl of `PDAsync` to the server creation, for this mockup, trait [`Case`](https://github.com/pingcap/tikv/pull/1591/files/32323494056ad8b557058774e5cbbc6086c3d8fe#diff-8c8b57642f7be45cd6688323ee907be6R32) is the real impl of `PDAsync`, and most of them need endpoints ahead.